examples/JsConsoleExample/JsConsoleProvider.fs.js

```diff
@@ -0,0 +1,137 @@
+import { some } from "./.fable/fable-library.3.2.12/Option.js";
+import { class_type } from "./.fable/fable-library.3.2.12/Reflection.js";
```

Comment:
Do we need to include any of the generated `.js` files?

Reply:
I don't think so, though I'll be honest I'm not 100% sure how distributing a fable library works :)

Reply:
if you're distributing a package then you do need to include the source files in a `js` folder of your package. This library is a source dependency though, and so consumers will already be getting the source, so that problem should be side-stepped.